### DISCUSSION TITLE
How to setup multilingual menu in astroid ?

### DISCUSSION THREAD

**Discussion by mikevanoostende** [templaza/astroid-framework#248] [Q&A]
I want to change the current template to an astroid based template, currently I have 4 languages for the menu, how
can i do this within astroid, it seems I can only select 1 menu ? 

thx, 
Mike.

**Comment by Chacapamac** [comment]
>  it seems I can only select 1 menu ?

 Yes, this is normal, you have to create a layout for each language/menus (4 Layout for 4 Languages)
— They are normally copy of your original layout and you have to assign each layouts to the correct Joomla menu in the **Template Style > Layout > Menu assignment**

![Template-Styles-Layouts](https://user-images.githubusercontent.com/1907276/202906726-d69f3845-f111-47a5-bd96-5f74b269d402.jpg)

![Assign-menus](https://user-images.githubusercontent.com/1907276/202906829-eb2afdd1-a03d-4b97-bf31-ac653d24e029.jpg)

**Reply by mikevanoostende** [reply]
Hi, thx for the quick answer, I'll dig into this !

regards, 
Mike.

**Reply by Chacapamac** [reply]
If you have any questions, I’m happy to help!
**Good Day!**

**Comment by paurray** [comment]
@Chacapamac Thank you this also helps me as I also want to do this.

**Comment by Chacapamac** [comment]
Happy to help!

Good Day!
